deploy/SLES15SP1/ntnx-csi-provisioner.yaml

```diff
@@ -24,7 +24,7 @@ spec:
       - name: csi-provisioner
         image: quay.io/k8scsi/csi-provisioner:v1.3.0
```

Comment:
same for SLES file

csi-provisioner and csi-resizer tag are not the same as centos deploy example

and all snapshot part is missing

Reply:
Updated csi-provisioner and csi-resizer versions. I will leave it to SUSE team to test and add support for snapshot.